### ISSUE TITLE
 Error executing action `restart` on resource 'windows_service[wuauserv]' - Timeout

### ISSUE THREAD

**Issue by anniehedgpeth** [criteo-cookbooks/wsus-client#43]
```
       Recipe: wsus-client::configure
         * registry_key[HKLM\Software\Policies\Microsoft\Windows\WindowsUpdate] action create
           - set value {:type=>:dword, :name=>"AcceptTrustedPublisherCerts", :data=>1}
           - set value {:type=>:dword, :name=>"DisableWindowsUpdateAccess", :data=>0}
           - set value {:type=>:dword, :name=>"ElevateNonAdmins", :data=>0}
           - set value {:type=>:string, :name=>"TargetGroup", :data=>""}
           - set value {:type=>:dword, :name=>"TargetGroupEnabled", :data=>0}
           - set value {:type=>:string, :name=>"WUServer", :data=>""}
           - set value {:type=>:string, :name=>"WUStatusServer", :data=>""}
         * windows_service[wuauserv] action restart
       
           ================================================================================
           Error executing action `restart` on resource 'windows_service[wuauserv]'
           ================================================================================
       
           Timeout::Error
           --------------
           Timeout::Error
       
           Resource Declaration:
           ---------------------
           # In C:/Users/jenkins_build_svc/AppData/Local/Temp/kitchen/cache/cookbooks/wsus-client/recipes/configure.rb
       
            97: service 'wuauserv' do
            98:   action :enable
            99:   retries 2
           100: end
           101: 
       
           Compiled Resource:
           ------------------
           # Declared in C:/Users/jenkins_build_svc/AppData/Local/Temp/kitchen/cache/cookbooks/wsus-client/recipes/configure.rb:97:in `from_file'
       
           windows_service("wuauserv") do
             action [:enable]
             default_guard_interpreter :default
             service_name "wuauserv"
             enabled nil
             running nil
             masked nil
             pattern "wuauserv"
             declared_type :service
             cookbook_name "wsus-client"
             recipe_name "configure"
             retries 2
           end
       
           System Info:
           ------------
           chef_version=14.10.9
           platform=windows
           platform_version=10.0.14393
           ruby=ruby 2.5.3p105 (2018-10-18 revision 65156) [x64-mingw32]
           program_name=C:/opscode/chef/bin/chef-client
           executable=C:/opscode/chef/bin/chef-client
       
       
       Running handlers:
       [2019-07-22T21:28:45+00:00] ERROR: Running exception handlers
       Running handlers complete
       [2019-07-22T21:28:45+00:00] ERROR: Exception handlers complete
       Chef Client failed. 25 resources updated in 14 minutes 25 seconds
       [2019-07-22T21:28:45+00:00] FATAL: Stacktrace dumped to C:/Users/jenkins_build_svc/AppData/Local/Temp/kitchen/cache/chef-stacktrace.out
       [2019-07-22T21:28:45+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
       [2019-07-22T21:28:45+00:00] FATAL: Timeout::Error: Timeout::Error
```

We keep getting these timeout errors only in test kitchen and only intermittently, so I'm thinking of creating a pull request that makes an attribute out of the timeout for the service. Thoughts?

**Comment by jeremy-clerc** [comment]
Fixed via #44

**Comment by jeremy-clerc** [comment]
2.0.3 released https://supermarket.chef.io/cookbooks/wsus-client#changelog

Thank you for your contribution.
